store/tikv/region_cache.go

```diff
@@ -1242,12 +1243,30 @@ func (c *RegionCache) getProxyStore(region *Region, store *Store, rs *RegionStor
 	}
 
 	tikvNum := rs.accessStoreNum(TiKVOnly)
-	for index := 0; index < tikvNum; index++ {
+	if tikvNum <= 1 {
+		return
```

Comment:
Does it mean to query the leader?

Reply:
`GetTiKVRPCContext` will get nil as the proxyStore, so no proxy will be used.